src/Enum/Type.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Yii\Bootstrap5\Enum;
+
+/**
+ * Is an enumeration of types for bootstrap5.
+ */
+enum Type: string
```

Comment:
Type is too common name. 

Bootstrap don't contain base types. They are different in different contexts. For exmaple, [text colors](https://getbootstrap.com/docs/5.3/utilities/colors/#colors) (24 types), [background color](https://getbootstrap.com/docs/5.3/utilities/background/#background-color) (22 types).

I suggest to separate type for each context as in Bootstrap 5. For alert it will be `AlertType`.

Reply:
All widgets use these types, only the class prefix changes, if we do it this way, we will repeat the code everywhere.

Reply:
Code repeat is OK for this case.